### DISCUSSION TITLE
[Solution][Conda error][CondaHTTPError: HTTP 000 CONNECTION FAILED]

### DISCUSSION THREAD

**Discussion by coskrrb2002** [idaholab/moose#19544] [Q&A Getting Started] [answered]
For those who suffer from the error of "CondaHTTPError: HTTP 000 CONNECTION FAILED".

Use the following steps.

1. Download the old version of Conda by the following command:
wget https://repo.anaconda.com/miniconda/Miniconda3-4.7.12.1-Linux-x86_64.sh
2. Install it.
bash Miniconda3-4.7.12.1-Linux-x86_64.sh -b -p ~/miniconda3
3. Then you can follow the guideline of moose for Linux:
 https://mooseframework.inl.gov/getting_started/installation/conda.html
Follow from the line: 
export PATH=$HOME/miniconda3/bin:$PATH

Cheers,
Nakkyu

**Comment by loganharbour** [comment, marked as answer]
I'm not sure I would suggest this. Errors like that usually indicative of an actual connection issue. 

Do you have a thread somewhere in which someone suggested a downgrade?

**Reply by coskrrb2002** [reply]
Here is the post:
https://github.com/conda/conda/issues/9948

I used a Ubuntu app for windows.

So, I was guessing that this connection issue is related to wsl version of windows.

By the way, after the installation of the old version of Conda, Conda can be updated.

So, until someone figure out the real problem, I guess this will be the temporary solution.

**Reply by milljm** [reply]
There are other work-arounds on that issue. Indeed I actually posted among that thread a couple months back.

This fix should work for any installation of Conda (Miniconda, Miniforge, Mambaforge, etc).

```bash
find miniconda3/ -type f -exec touch {} +
```

I prefer the above solution over others mentioning `chmod -R 777`. For obvious security reasons.
